### PR TITLE
[Desktop] Release notes for 1.18.x Release #2

### LINKED ISSUE #13247: [Desktop] Release Notes for 1.18.x Release #2 (Release Channel)
### Release Notes:

- Ads per hour frequency cap incorrectly limits based on adsPerHour not adsPerDay. ([#13215](https://github.com/brave/brave-browser/issues/13215))
- NTP is causing higher CPU / GPU usage. ([#13194](https://github.com/brave/brave-browser/issues/13194))
- Display iOS QR Code feature on Desktop. ([#13159](https://github.com/brave/brave-browser/issues/13159))

### PR DESCRIPTION
**Pull request by LaurenWags**
Fix #13247

## Submitter Checklist:

- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed).
- [ ] Requested a security/privacy review as needed.

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

n/a

### CODE REVIEW

File: CHANGELOG_DESKTOP.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [1.18.75](https://github.com/brave/brave-browser/releases/tag/v1.18.75)
+
+ - Added QR code under brave://rewards for iOS devices. ([#13159](https://github.com/brave/brave-browser/issues/13159))
+ - Fixed issue where the New Tab Page was causing persistently higher CPU usage. ([#13194](https://github.com/brave/brave-browser/issues/13194))
+ - Fixed issue where the number of Brave Ads delivered per day was incorrectly being limited to the ads per hour selection. ([#13215](https://github.com/brave/brave-browser/issues/13215))
```
Comment (rebron) [inline CHANGELOG_DESKTOP.md:7]: "Limited to" -> "limited by"

**Review by rebron — COMMENTED** [review]
other than that one, looks good.

File: CHANGELOG_DESKTOP.md (hunk 1 above)
Comment (LaurenWags) [reply CHANGELOG_DESKTOP.md]: updated

**Review by rebron — APPROVED** [review]
Looks good. ++

File: CHANGELOG_DESKTOP.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [1.18.75](https://github.com/brave/brave-browser/releases/tag/v1.18.75)
+
+ - Added QR code under brave://rewards for iOS devices. ([#13159](https://github.com/brave/brave-browser/issues/13159))
+ - Fixed issue where the New Tab Page was causing persistently higher CPU usage. ([#13194](https://github.com/brave/brave-browser/issues/13194))
```
Comment (kjozwiak) [inline CHANGELOG_DESKTOP.md:6]: We should add GPU to this sentence as it affected both. Example:

```
- Fixed issue where the New Tab Page was causing persistently higher CPU and GPU usage.
```

**Review by kjozwiak — CHANGES_REQUESTED** [review]

File: CHANGELOG_DESKTOP.md (hunk 2 above)
Comment (LaurenWags) [reply CHANGELOG_DESKTOP.md]: updated

**Review by kjozwiak — APPROVED** [review]
LGTM 👍

**Pull request merged**